pkg/network/setup/netstat.go

```diff
@@ -205,3 +219,14 @@ func newVMIIfaceStatusFromGuestAgentData(guestAgentInterface api.InterfaceStatus
 		InterfaceName: guestAgentInterface.InterfaceName,
 	}
 }
+
+func filterHostDevicesByAlias(hostDevices []api.HostDevice, prefix string) []api.HostDevice {
+	var filteredHostDevices []api.HostDevice
+
+	for _, hostDevice := range hostDevices {
+		if hostDevice.Alias != nil && strings.HasPrefix(hostDevice.Alias.GetName(), prefix) {
+			filteredHostDevices = append(filteredHostDevices, hostDevice)
+		}
+	}
+	return filteredHostDevices
+}
```

Comment:
Same reason as before, we cannot depend on virt-launcher.

Reply:
But the other way is possible (using pkg/network/setup/netstate.go filterHostDevicesByAlias), right?

Reply:
What about `filterHostDevicesByAlias` can we move it to a common place?

Reply:
What about filterHostDevicesByAlias can we move it to a common place?

Reply:
I think it fits under something equivalent to `network/vmispec`, unfortunately, someone already used `domainspec` for other stuff. I prefer not to just place it there without understanding well if the current content there fits.

If you have other suggestions, please mention it.

Reply:
From what I saw `domainspec` handle the network interfaces part of the domain XML, 
does it mean that it cant have other stuff related to the domain spec?
Since `filterHostDevicesByAlias` generic form, it can also fit its own package outside `pkg/network` something like `pkg/device`.
Deciding to which package it should go shouldn't distract us from the fact that either way its better than have it in two places.